src/zenml/zen_stores/rest_zen_store.py

```diff
@@ -496,674 +514,929 @@ def get_deployment_id(self) -> UUID:
         """
         return self.get_store_info().id
 
-    # ------
-    # Stacks
-    # ------
+    # ----------------------------- API Keys -----------------------------
```

Comment:
A side of me loves that these blocks are sorted alphabetically now. The other side really doesn't and thinks you should not have done it in a PR that is already this huge. I can't review this file.

Reply:
I wasn't expecting things to get out of hand at this scale. Will be more considerate of this in the future. Thank you for the tough review in any case. <3